README.md

```diff
@@ -7,22 +7,301 @@ Serlio requires so-called rule packages (RPK) as input, which are authored in Ci
 Serlio is well suited for managing the procedural generation of architectural 3D content in digital sets. However, Serlio is restricted to the procedural generation of single buildings / objects. Serlio does not include the city layouting and street network editing tools of CityEngine i.e. the rich CityEngine toolset to design a city from scratch (or based on geographic data) is still needed.
 
 
-## Documentation
+## Quick Start
```

Comment:
I would limit the Quick Start to Windows and the installer, i.e. in the second step just write "Please see below for installation on Linux."). The quick start should be as concise as possible.

Reply:
limited the quick start guide to windows.